src/pages/account/logins/[type]/[id].tsx

```diff
@@ -46,15 +44,19 @@ export default IdentityOverviewPage;
 export const getServerSideProps: GetServerSideProps = async (context) => {
   return wrapMiddlewaresForSSR<{ type: string }>(
     context,
-    [withLogger, withSentry, withSession, withAuth],
-    async (request: ExtendedRequest, response: ServerResponse) => {
+    [withLogger, withSentry, withAuth],
+    async (request, response) => {
       if (!context?.params?.id) {
         response.statusCode = 400;
         response.end();
         return;
       }
 
-      const userCookie = request.session.get<UserCookie>("user-cookie");
+      const userCookie = await getServerSideCookies<UserCookie>(
+        request,
+        "user-cookie",
+        true,
```

Comment:
I still find this `true` weird when reading it 😅

Reply:
Oh damn, I forgot to give it as an object. I'll change it.